add toolbar image

baezluise opened the pull request:
I have added the files and images for the header. I also created the headline element.

baezluise committed:
add toolbar image

baezluise committed:
add social media icons and text

baezluise committed:
edit README

baezluise committed:
fix linter errors

baezluise committed:
fix indentation issues

baezluise committed:
fix two indetation problems

baezluise commented:
I have added the files and created the header and headline element.
Also used the figma template properties for CSS file.

GedeonTS commented on index.html:
```diff
@@ -0,0 +1,28 @@
+</head>
+<body>
+  <header>
+    <div id="welcome"><a>Welcome</a></div>
```
- [x] Good job using an anchor tag :+1: , Kindly make sure whenever you use it, it has a href="#" attribute so that it can lead somewhere :recycle:

GedeonTS commented on index.html:
```diff
@@ -0,0 +1,28 @@
+    <div id="welcome"><a>Welcome</a></div>
+    <nav id="nav"><a><img src="hamburger.png"></a></nav>
+  </header>
+  <headline>
```
- [x] :thinking:  In my opinion, the headline tag does not exist, so kindly use a section tag for semantic purpose :recycle:

GedeonTS commented on index.html:
```diff
@@ -0,0 +1,28 @@
+    <nav id="nav"><a><img src="hamburger.png"></a></nav>
+  </header>
+  <headline>
+    <div id="background"><h2 class="text-container" id="hisection">Hey there. I’m Luis<h2 class="text-container" id="sdsection">I’m a software developer</h2></h2>
```
- [x] It would be better to have a h1 title before h2 and h3 to respect the HTML semantic rules, so kindly make sure the first one is h1 :recycle:

GedeonTS commented on index.html:
```diff
@@ -0,0 +1,28 @@
+  <headline>
+    <div id="background"><h2 class="text-container" id="hisection">Hey there. I’m Luis<h2 class="text-container" id="sdsection">I’m a software developer</h2></h2>
+      <h3 id="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</h3></div>
```
- [x] As per the project requirements please make sure this text is a paragragrap <p> :recycle:

GedeonTS commented on index.html:
```diff
@@ -0,0 +1,28 @@
+        <div class="social-media"><img src="medium.svg"></div>
+        <div class="social-media"><img src="github.svg"></div>
+        <div class="social-media"><img src="angellist.svg"></div>
+      </li>
```
- [x] Good Idea trying to use a list :clap: , however, the right syntax is as follow:

`<ul>`
`<li><a href="#"><img src="icon1"></a></li>`
`<li><a href="#"><img src="icon2"></a></li>`
.....
`</ul>`

GedeonTS commented on styles.css:
```diff
@@ -0,0 +1,53 @@
+}
+
+.text-container {
+  font-style: normal;
```
- [x] Kindly make sure to use the right font as we have to stick to the design, and use flexbox for contents( __HINTS:__ font-family:Inter; display: flex; flex-direction: column) :recycle:

GedeonTS commented on styles.css:
```diff
@@ -0,0 +1,53 @@
+
+header {
+  align-content: space-between;
+}
```
- [ ] :thinking:  In order to allow the `align-content: space-between;` to work, kindly make sure to precede it with a `display:flex` and a `width:100%` :recycle: 
![Yheader](https://user-images.githubusercontent.com/97834160/212233814-8645e5d5-eea0-45cb-aabf-7e4ebaaaf77a.png)

GedeonTS commented on styles.css:
```diff
@@ -0,0 +1,53 @@
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
```
- [ ] Kindly make sure your design looks like the figma design and contains all the present elements. :recycle: 
- Your design:
![Screenshot from 2023-01-13 05-57-31](https://user-images.githubusercontent.com/97834160/212234099-9d8e9f5e-0ea0-4933-a138-5d967dffe908.png)
- Figma:
![Fbottom](https://user-images.githubusercontent.com/97834160/212234337-0cc713df-3bfc-43f6-bdf4-a308183c0660.png)

GedeonTS reviewed (changes requested):
# STATUS: CHANGES REQUIRED :recycle: :

Hi @baezluise , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif">

## Highlights :tophat:  :+1: 

- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- Good README :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] In my opinion it should be better to write more meaningfull commit messages to make your work more professional (starting with a capital letter, using the emperative form and describing clearly the change you made) :recycle: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baezluise committed:
correct list structure and add family font

I changed the structure of the icons list and I've added the font-family along with the display-flex.

baezluise committed:
add font source

baezluise committed:
style button and icons

-changed padding to close gaps between icons.
-arrow button style

baezluise committed:
Update styles CSS.

baezluise committed:
fix font family error

baezluise committed:
fix font family

i added sans-serif as font- generic

baezluise commented:
I have added right fonts, icons missing and center elements.
I have created the list structure and added href to anchor tags.

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,42 @@
+</head>
+<body>
+  <header>
+    <div id="welcome" href="#">Welcome</div>
```
- [x] I can see you tried to implement the suggested changes 👍🏽. Kindly note that `href` is only applied to anchor` <a>` tag and not a `div` tag. KIndly fix it 👍🏽 

e.g `<a href="#">Welcome </a>`

victorokoroji commented on styles.css:
```diff
@@ -0,0 +1,85 @@
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  align-content: space-between;
+  justify-content: space-between;
+}
```
- [ ] The content of your header is not properly aligned vertically at the center because it is missing the property `align-item: center`. KIndly replace align-content with it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/212425813-80383b1c-5124-4627-a670-8479a1c19b63.png)

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,42 @@
+      </h1>
+      <p id="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p></div>
+      <ul>
+        <li><a href="#"><img src="twitter.svg"></a></li>
+        <li><a href="#"><img src="linkedin.svg"></a></li>
+        <li><a href="#"><img src="medium.svg"></a></li>
+        <li><a href="#"><img src="github.svg"></a></li>
+        <li><a href="#"><img src="angellist.svg"></a></li>
+      </ul>
```
- [ ] KIndly align your social icons to be on the same line so that it matches the figma design just as the previous reviewer has suggested 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212426379-0c34c508-8b59-4875-a662-16be6ab8cc3e.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212426562-f5b2d061-30da-4ed5-9cb9-57054befeb8e.png)

victorokoroji reviewed (changes requested):
Hi @baezluise, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baezluise committed:
aling menu button

baezluise committed:
add .gitignore and update styles

baezluise commented:
menu aligned with welcome 
delete href in welcome
aligned icons

Felix45 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+</head>
+<body>
+  <header>
+    <div id="welcome">Welcome</div>
```
- [ Optional ] Please consider wrapping the text `Welcome` in an `a` tag instead of a `div` tag, since the text serves as a logo it's best to wrap it in an `a` tag like is the case in most websites and for User Experience when navigating your webpage.

Felix45 reviewed (approved):
# Approved :green_circle: 
Hi @baezluise , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/53e1017a622c054f387c8c4e72208e5b65b81c4e38605bd2fd8a0391467f6ace/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f42504a6d74685133595277443651716356442f67697068792e676966)

## Highlights :heavy_check_mark: 
- [ ] Implemented all the requested changes from the previous code reviewer :heavy_check_mark: 
- [ ] GitHub flow is used correctly :heavy_check_mark: 
- [ ] All linter checks are set up and passing with green checks :heavy_check_mark: 
- [ ] Professional readme file :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
